src/mame/drivers/jpmsru.cpp

```diff
@@ -65,6 +65,14 @@
 #include "j_ndu.lh"
 #include "j_dud.lh"
 #include "j_lan.lh"
+#include "j_super2.lh"
+#include "j_ews.lh"
+#include "j_ewsdlx.lh"
+#include "j_ssh.lh"
+#include "j_lt.lh"
+#include "j_plus2.lh"
+#include "j_sup2p.lh"
+#include "j_la.lh"
```

Comment:
It’s better to sort these so people can see what’s here at a glance.

Reply:
Fixed.